### ISSUE TITLE
Emoji ⚓️Will Display a Black X with a square

### ISSUE THREAD

**Issue by CoelWu** [samhocevar/emoji.wpf#47]
When I use this emoji ⚓️, A black square(or sometimes with X in it) will be displayed after it.

Emoji.WPF Sample Program
![image](https://user-images.githubusercontent.com/76829607/128596189-ca88a740-6b06-4dd5-84bd-e6f120707d6c.png)

My Program which use emoji.wpf's TextBlock:
![image](https://user-images.githubusercontent.com/76829607/128596231-39a25a24-972b-4420-a5c4-02a491c2d9e0.png)

**Comment by samhocevar** [comment]
Yes, this is a bug (same as issue #18). It happens because there are two ways to write the Anchor emoji in Unicode: either `U+2693` alone (⚓), or `U+2693 U+FE0F` (⚓️). They look exactly the same, but for the moment Emoji.Wpf is unable to render the latter without that extra rectangle for now.

I’ll have a closer look at this issue soon.
